interfaces/cython/cantera/onedim.py

```diff
@@ -27,7 +22,7 @@ def __init__(self, domains, gas, grid=None):
         if grid is None:
             grid = np.linspace(0.0, 0.1, 6)
         self.flame.grid = grid
-        super(FlameBase, self).__init__(domains)
+        super().__init__(domains)
```

Comment:
👏 Here's one of the Python 3 features I didn't think of, that is super useful, if only in making it easier for new people to jump in without having to know which arguments are which, etc.

Reply:
This is one of my favorite little clean-ups in Python 3, removing the need to specify completely redundant information.